config/routes.rb

```diff
@@ -24,6 +24,7 @@
   resources :users
   resources :teams
   resources :splash
+  resources :mobile_dashboard
```

Comment:
Still required.  Games Controller uses this route in order to access mobile_dashboard if you are player.